app/Policies/UserPolicy.php

```diff
@@ -114,6 +115,25 @@ public function view(User $user, User $target): bool
         return false;
     }
 
+    /**
+     * Whether one can edit the user's educational or personal information.
+     *
+     * @param User $user
+     * @param User $target
+     * @return bool
+     */
+    public function updateInformation(User $user, User $target): bool
+    {
+        if (!$target->verified && $target->isCollegist()) {
+            // that means, they are applicants
```

Comment:
not at all.
- a verified user can still be an applicant
- new applicants don't have collegist role from now on

Editing data for applicants is not yet implemented anyways, I would just delete this for now.